index.html

```diff
@@ -63,7 +63,110 @@ <h4>Skills</h4>
             <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
             <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
         </div>
-        <hr>                    
+        <span>&#8595; &nbsp;&nbsp; WORK</span>
+        <div class="works-flex">
+            <h1>MY RECENT WORK</h1>
+            <br>
+            <hr>
+            
+            <div class="project-flex">
+                <div class="content-flex">
+                    <div class="layout-flex">
+                        <div>
+                            <img src="./" alt="project-1">
```

Comment:
None of your projects have images, please refer to the Figma template and add blank images.

<img width="532" alt="Screen Shot 2022-07-07 at 21 43 15" src="https://user-images.githubusercontent.com/82426682/177906427-c8c69934-c1b4-4ee6-9368-1109c8ff3554.png">

Reply:
The project image has been used as per Figma template.